### PR TITLE
TDR-1965 Rewording of error page and add reference to the email subject

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: app/views/partials/fileCheckGenericErrorMessage.scala.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
+@(consignmentRef: String)
 <p class="govuk-body">
-    One or more files you uploaded have failed our checks. Please
-    <a class="govuk-link" href="mailto:tdr@@nationalachives.gov.uk?subject=Problem with Results of checks">Contact us</a>
+    One or more files you uploaded have failed our checks. Contact us at
+    <a class="govuk-link" href="mailto:tdr@@nationalachives.gov.uk?subject=Ref: @consignmentRef Problem with Results of checks">tdr@@nationalachives.gov.uk</a>
```
Comment (techncl) [inline app/views/partials/fileCheckGenericErrorMessage.scala.html:4]: I was thinking that maybe there should be a hyphen in-between the "consignmentRef" and the "Problem with Results of checks" like `@consignmentRef  - Problem with Results of checks` just for clarity.

**Review by techncl — COMMENTED** [review]
Looks good. There was just one comment about adding a hyphen. Also, I was thinking maybe the link could be added to the unit tests, just to make it more robust?

File: app/views/partials/fileCheckGenericErrorMessage.scala.html (hunk 1 above)
Comment (thanhz) [reply app/views/partials/fileCheckGenericErrorMessage.scala.html:4]: Yeah that would make it easier to read :+1:

**Comment by thanhz** [comment]
After discussion with Kirsten we have decided to word the error as such **"One or more files you uploaded have failed our checks. Contact us at  [tdr@nationalarchives.gov.uk](mailto:tdr@nationalarchives.gov.uk) if the problem persists."**

**Review by techncl — APPROVED** [review]

**Pull request merged**